### PR TITLE
レビューをIDの降順で取得する

### PR DESCRIPTION
**Pull request by yamada-kensyu**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/repository/ReviewRepository.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public Review load(Integer id) {
 	 * @return　レビューのリスト
 	 */
 	public List<Review> findByItemId(Integer itemId) {
-		String sql = "Select id, rating, title, comment, item_id, user_id from reviews WHERE item_id=:itemId;";
+		String sql = "Select id, rating, title, comment, item_id, user_id from reviews WHERE item_id=:itemId ORDER BY id DESC;";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ReviewRepository.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 136).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ReviewRepository.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ReviewRepository.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

**Review by quangtung98 — APPROVED** [review]
問題がなさそうと思いますので、承認します。
修正ありがとうございました

**Pull request merged**